### PR TITLE
CC-24698: Fixed discontinued product attributes label.

### PR DESCRIPTION
**Pull request by sakharova-yuliia**
#### Overview

- Ticket: https://spryker.atlassian.net/browse/CC-24698
-  Integration Ticket: https://spryker.atlassian.net/browse/CC-26349

### CODE REVIEW

File: src/Pyz/Yves/ProductDetailPage/Theme/default/components/molecules/variant-configurator/variant-configurator.twig [hunk 1]
```diff
@@ -25,6 +25,12 @@
         {% set selectedValue = data.selectedAttributes[name] | default %}
         {% set selectedValue = selectedValue in values ? selectedValue : null %}
         {% set isAvailable = true %}
+        {% set optionResetUrl = '' %}
+
+        {% for selectedAttributeName, selectedAttributeValue in data.selectedAttributes | filter((value, key) => key != name) %}
+            {% set selectedUrlParameter = [('attribute[' ~ selectedAttributeName ~ ']')| url_encode, selectedAttributeValue] | join('=') %}
```
Comment (danilmoro3) [inline src/Pyz/Yves/ProductDetailPage/Theme/default/components/molecules/variant-configurator/variant-configurator.twig:31]:
```suggestion
            {% set selectedUrlParameter = [('attribute[' ~ selectedAttributeName ~ ']') | url_encode, selectedAttributeValue] | join('=') %}
```

**Review by danilmoro3 — APPROVED** [review]

**Review by FRITS-Kh — CHANGES_REQUESTED** [review]
The same comments https://github.com/spryker-shop/b2b-demo-shop/pull/143#pullrequestreview-1352952267

**Review by ilyakubanov — CHANGES_REQUESTED** [review]
- please revert downgrading the modules versions (not shown as diff but the version in integration branch is downgraded)

File: src/Pyz/Yves/ProductDetailPage/Theme/default/components/molecules/variant-configurator/variant-configurator.twig [hunk 2]
```diff
@@ -25,6 +25,12 @@
         {% set selectedValue = data.selectedAttributes[name] | default %}
         {% set selectedValue = selectedValue in values ? selectedValue : null %}
         {% set isAvailable = true %}
+        {% set optionResetUrl = '' %}
+
+        {% for selectedAttributeName, selectedAttributeValue in data.selectedAttributes | filter((value, key) => key != name) %}
+            {% set selectedUrlParameter = ('attribute[' ~ selectedAttributeName ~ ']')| url_encode ~ '=' ~ selectedAttributeValue %}
```
Comment (FRITS-Kh) [inline src/Pyz/Yves/ProductDetailPage/Theme/default/components/molecules/variant-configurator/variant-configurator.twig:31]:
```suggestion
            {% set selectedUrlParameter = ('attribute[' ~ selectedAttributeName ~ ']') | url_encode ~ '=' ~ selectedAttributeValue %}
```

**Review by FRITS-Kh — APPROVED** [review]

**Review by ilyakubanov — APPROVED** [review]

**Pull request merged**